.github/dependabot.yml

```
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: github-actions
     directory: "/"
     schedule:
-      interval: daily
+      interval: weekly
```

```suggestion
      interval: weekly
    commit-message:
      prefix: "chore(deps)"
    groups:
      dependencies:
        applies-to: version-updates
        update-types:
          - "minor"
          - "patch"
```

like this @jmeridth ?